k8s/operators/src/lib.rs

```diff
@@ -2,6 +2,6 @@
 pub mod diskpool {
     /// The DiskPool custom resource definition.
     pub mod crd {
-        include!("pool/diskpool/crd.rs");
+        include!("pool/diskpool/v1alpha1.rs");
```

Comment:
This is for exposing the CRD from the library, shouldn't this be v1beta1? Otherwise we're exposing the older version.